pkg/deploy/assets/databases-development.json

```diff
@@ -11,160 +11,146 @@
     },
```

Comment:
The JSON diff is quite confusing. Was it sorted and if so can you revert that?

Huge +1 alphabetical sorting but in a separate PR

Reply:
The json is generated programatically with `make generate`. I think it isn't specifically sorted.